docs/_static/auth/iam.yaml

```diff
@@ -238,6 +275,35 @@ components:
           "token_type": "bearer",
           "expires_in": 3600,
         }
+    OAuthAuthorizationServerMetadata:
```

Comment:
if we don't even generate code from this, just make it an `object`

Reply:
I removed the content, but it is still there as a schema since removing it completely makes the codegenerator assign it type `map[string]interface{}` which cannot be aliased to the actual internal struct.